### PR TITLE
replaced react logo with temporary logo, and added welcome message

### PR DESCRIPTION
**Pull request by lx93**
with username displayed up front
logo

### CODE REVIEW

File: src/client/src/App.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ class App extends Component {
         return (
           <div className="App">
             <header className="App-header">
-              <img src={logo} className="App-logo" alt="logo" />
+              <img src={logo} alt="logo" />
+	      <p>Welcome to ChatNow, {JSON.stringify(this.state.user.name)}</p>
```
Comment (imann24) [inline src/client/src/App.js:81]: Is it necessary to `stringify`?

File: src/client/src/App.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import logo from './logo.png';
```
Comment (imann24) [inline src/client/src/App.js:2]: Love the new logo!

If you don't mind taking it an extra step, would be great to change the favicon and page title too: 
image

**Review by imann24 — APPROVED** [review]
Excellent visual polish!

File: src/client/src/App.js (hunk 1 above)
Comment (lx93) [reply src/client/src/App.js:81]: believe so otherwise it will just show [Object object]

**Comment by lx93** [comment]
deleting this pull request and branch. starting over

**Pull request closed without merging**